exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/BitToUserConnectionIterator.java

```diff
@@ -99,14 +102,18 @@ public void remove() {
 
   public static class ConnectionInfo {
     public String user;
+    @Nonnull
```

Comment:
Nice use of the Java native annotation!

Reply:
Thanks for pointing out the availability of these annotations. Didn't want to reinvent the wheel. :)

Reply:
The thing is that this is not native java annotation. It's absent in java 8 SE specification.
https://docs.oracle.com/javase/8/docs/api/javax/annotation/package-summary.html
http://mvnrepository.com/artifact/javax.annotation/javax.annotation-api/1.3.2

Reply:
That is odd. I actually was referring to this: https://blogs.oracle.com/java-platform-group/java-8s-new-type-annotations
I suspect the documentation is missing, because the `javax.annotation.Nonnull` is meant for runtime code generation
```javax.annotation.Nonnull

@TypeQualifier
@Retention(value=RUNTIME)
@Documented
```

Reply:
Ok. Got it. This is part of the annotations-2.0.1 dependency that drill has in its 3rd party library (`com.google.code.findbugs`). It might be used here in a context different from what it was intended to be used for.
So, @arina-ielchiieva / @paul-rogers , should I go ahead with this usage or revert to implementing the annotation class I originally had?

Reply:
Sorry, I was misled by the "javax" package and thought this was a Java thing. Good catch @arina-ielchiieva.

Since it is not, go ahead and define your own. Today we assume all fields are nullable, and must keep that default. So, since the default is nullable, you only need the `@NonNullable` annotation.

Reply:
+1 for @NotNullable